### PR TITLE
For #14268: Implement ob=

### PR DESCRIPTION
**Pull request by deepakchethan**
Output:

Screen Shot 2019-06-10 at 3 37 29 PM

### CODE REVIEW

File: libr/core/cmd_open.c [hunk 1]
```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
 			}
 		}
 		break;
+	case '=': // "ob="
+		{
+			RListIter *iter;
+			RList *list = r_list_new ();
+			RBinFile *binfile = NULL;
```
Comment (radare) [inline libr/core/cmd_open.c]: Use bf instead of binfile. its shorter and im doing this change in all the codebase

File: libr/core/cmd_open.c [hunk 2]
```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
...
+			RBin *bin = core->bin;
+			const RList *binfiles = bin ? bin->binfiles: NULL;
```
Comment (radare) [inline libr/core/cmd_open.c]: unnecessary localvar

File: libr/core/cmd_open.c [hunk 3]
```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
...
+			if (!binfiles) {
```
Comment (radare) [inline libr/core/cmd_open.c]: should never be null, so r_lsit_foreach will assert, unnecessary if

File: libr/core/cmd_open.c [hunk 4]
```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
...
+				return;
+			}
+			r_list_foreach (binfiles, iter, binfile) {
+				char temp[4];
+				ListInfo *info = R_NEW (ListInfo);
+				if (!info) {
+					return;
```
Comment (radare) [inline libr/core/cmd_open.c]: break instead of return

File: libr/core/cmd_open.c [hunk 5]
```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
 			}
 		}
 		break;
+	case '=': // "ob="
+		{
+			RListIter *iter;
+			RList *list = r_list_new ();
```
Comment (radare) [inline libr/core/cmd_open.c]: r_list_newf(free), but should be (r_listinfo_free),

File: libr/core/cmd_open.c [hunk 6]
```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
...
+			const RList *binfiles = bin ? bin->binfiles: NULL;
+			if (!binfiles) {
+				return;
+			}
+			r_list_foreach (binfiles, iter, binfile) {
+				char temp[4];
+				ListInfo *info = R_NEW (ListInfo);
+				if (!info) {
+					return;
+				}
```
Comment (radare) [inline libr/core/cmd_open.c]: maybe good to have an r_listinfo_new() to just pass 4 parameters instead of doing the new+struct init by hand

Comment (deepakchethan) [reply libr/core/cmd_open.c]: Okay :)

**Pull request merged**